src/Avalonia.Base/PropertyStore/ValueStore.cs

```diff
@@ -209,6 +209,20 @@ public void ClearLocalValue(AvaloniaProperty property)
             }
         }
 
+        public void SetCurrentValue<T>(StyledProperty<T> property, T value)
+        {
+            if (_effectiveValues.TryGetValue(property, out var v))
```

Comment:
`TryGetEffectiveValue` is used everywhele else, even in `ClearLocalValue`. It seems like a redundant wrapper, though?

Reply:
Using `TryGetEffectiveValue` here too now for consistency.